File: src/main/java/com/godwin/jsonparser/ui/ParserToolWindowPanel.java
package com.godwin.jsonparser.ui;

import com.intellij.ide.util.PropertiesComponent;
import com.intellij.openapi.ui.SimpleToolWindowPanel;
import com.intellij.openapi.wm.ToolWindow;

/**
 * Created by Godwin on 4/21/2018 12:32 PM for json.
 *
 * @author : Godwin Joseph Kurinjikattu
 */
public class ParserToolWindowPanel extends SimpleToolWindowPanel {
    private PropertiesComponent myPropertiesComponent;
    private ToolWindow myWindow;

    public ParserToolWindowPanel(PropertiesComponent propertiesComponent, ToolWindow window) {
        super(false, true);
        myPropertiesComponent = propertiesComponent;
        myWindow = window;
    }
}

File: src/main/kotlin/com/godwin/jsonparser/ui/ParserToolWindowPanel.kt
package com.godwin.jsonparser.ui

import com.intellij.ide.util.PropertiesComponent
import com.intellij.openapi.ui.SimpleToolWindowPanel
import com.intellij.openapi.wm.ToolWindow

class ParserToolWindowPanel(
    private val propertiesComponent: PropertiesComponent,
    private val window: ToolWindow
) : SimpleToolWindowPanel(false, true)
